src/cli/command-package.ts

```diff
@@ -34,6 +37,17 @@ export default async function run(options: CliOptions) {
     process.exit(1);
   }
 
+  for (const rawMapping of options['dependency-mapping']) {
```

Comment:
can you move this whole thing into the parse function?

Reply:
I had this in an earlier version, but I liked the separation of concerns as-is. One method parses, one method saves, and then this for loop iterates it all over our CLI-provided options array. 

Happy to move this all into an additional `processAllDependencyMappings` function if you feel strongly here. Since you approved this I'll assume it's okay to merge as-is, but just ping me here / on chat and I'll add this in a separate PR.